package/libhelix-aac/libhelix-aac.mk

```diff
@@ -10,7 +10,7 @@ LIBHELIX_AAC_INSTALL_TARGET = YES
 define LIBHELIX_AAC_BUILD_CMDS
 	$(foreach src, $(wildcard $(@D)/src/libhelix-aac/*.c), \
 		$(TARGET_CC) $(TARGET_CFLAGS) -I$(@D)/src/libhelix-aac -fPIC -DUSE_DEFAULT_STDLIB -c $(src) -o $(patsubst %.c, %.o, $(src));)
-	find $(@D)/src/libhelix-aac -type f -name *.o | xargs $(TARGET_CC) $(TARGET_LDFLAGS) -shared -o $(@D)/libhelix-aac.so
+	find $(@D)/src/libhelix-aac -type f -name '*.o' | xargs $(TARGET_CC) $(TARGET_LDFLAGS) -shared -o $(@D)/libhelix-aac.so
```

Comment:
This caused my build to break when I added the esp_microspeech_features library. No idea why. The small changes fixes it though.